js/graphql.md

```diff
@@ -0,0 +1,1374 @@
+---
+---
```

Comment:
Can we add a TOC at the top so that people can quick link direct to the specific directive they want to read about?